distros/fedora/fedora-install.sh

```
@@ -0,0 +1,969 @@
+        if dnf list --available uv &>/dev/null; then
+            log_info "Installing uv via dnf..."
+            if sudo dnf install -y uv; then
+                log_success "uv installed via dnf"
+            else
+                log_warn "Failed to install uv via dnf; will attempt manual install"
+            fi
+        fi
+
+        if ! command -v uv &>/dev/null; then
+            log_info "Installing uv from verified release artifact..."
+            local arch triple target url sum_url tmpbin
```

Shellcheck (0.11.0)

[warning] triple appears unused. Verify use (or export if used externally).

(SC2034)